packages/commercetools/api-client/docs/docs.api.md

```diff
@@ -0,0 +1,130 @@
+## API Report File for "@vue-storefront/docs"

```

Comment:
i think it shoudl be called api-client, not docs

Reply:
Actually i have no idea how to change this, configuring the api-extractor...